packages/filebrowser/src/listing.ts

```diff
@@ -1142,11 +1136,54 @@ export class DirListing extends Widget {
       }
     }
 
-    // Write to DOM
```

Comment:
Let's keep this comment :) It is important to separate reading and writing to DOM steps for performance reasons.